src/app/pages/forgot-password/forgot-password.component.html

```diff
@@ -1 +1,10 @@
-<p>forgot-password works!</p>
+<header class="header">
```

Comment:
looks like h1-h2, but not like header

Reply:
@valeriya-reva 
Do you mean I should change <p> to <h1> and second to <h2>?

Reply:
@dmitry-darly-solutions 
Fixed.